simapp/go.mod

```diff
@@ -0,0 +1,213 @@
+go 1.22.2
+
+toolchain go1.22.3
+
+module github.com/cosmos/ibc-go/simapp
+
+replace (
+	github.com/cosmos/ibc-go/modules/capability => ../modules/capability
+	github.com/cosmos/ibc-go/v8 => ../
+)
```

Comment:
I'm not sure how this is going to work with compatibility tests. I think this is fine for PR e2e tests etc.

It could become messy if we are building images with the latest each library etc.. 
Is the plan to pin these to specific releases when building an actual simapp version image?
 e.g. `ibc-go-simd:v9.0.0`

Reply:
I just assumed that was what we were already doing?

Reply:
with the compatibility tests we are basically just using the regular simd image I believe. Not using the wasm/callbacks binaries. 

I think this is what we are actually already doing, it's not ideal but I think until it becomes an actual problem it is okay to leave as is

Reply:
So they idea will be that we tag a new release of this module with every ibc-go major (and minor?) release?

Reply:
I don't think we need to tag or release the simapp go mod via github, but having a docker image built and pushed would be a good idea!

Reply:
Don't we already do that (we tag the image on release, no)? Maybe I am missing something, but since we use replace in go.mod, I am not sure why this would be any different from what we have today?